pyvisa_py/tcpip.py

```diff
@@ -1099,3 +1107,40 @@ def _set_attribute(
 
         """
         raise UnknownAttribute(attribute)
+
+
+def get_services(service_type, wait_time=0.1):
+    if zeroconf is None:
+        raise NotImplementedError(
+            "zeroconf package not found...try 'pip install zeroconf'"
+        )
```

Comment:
```suggestion
        raise NotImplementedError(
            "HiSLIP instrument discovery requires the zeroconf package to be installed... "
            "try 'pip install zeroconf'"
        )
```

Reply:
We could also add an optional dependencies in pyproject.toml.

`hislip-discovery= ["zeroconf"]`